File: Programming Language(ver.2)/C#/505.cs
using System;

public class TQC505
{
    public static void Main()
    {
        double a = Convert.ToDouble(Console.ReadLine());
        double b = Convert.ToDouble(Console.ReadLine());
        double c = Convert.ToDouble(Console.ReadLine());
        double d = Convert.ToDouble(Console.ReadLine());
        double e = Convert.ToDouble(Console.ReadLine());
        double f = Convert.ToDouble(Console.ReadLine());
        double result = 0;
        result = Math.Abs(a) * Math.Floor(b);
        result += Math.Pow(c, d) * Math.Sqrt(e);
        result += Math.Log10(f);
        string r = result.ToString("0.00");    
        Console.WriteLine(r);
    }
}

File: Programming Language(ver.2)/C/505.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main ()
{
	double result;
	double a, b, c, d, e, f;
	scanf("%lf %lf %lf", &a, &b, &c);
	scanf("%lf %lf %lf", &d, &e, &f);
	
	result = fabs(a)*floor(b)+pow(c, d)*sqrt(e)+log10(f);
	int result_2 = (result+0.005)*100;
	
	printf("%.2f\n", result_2/100.);
    return 0;
} 
